### DISCUSSION TITLE
SQL Profiler Greyed Out

### DISCUSSION THREAD

**Discussion by briannewtonpsyd** [DaxStudio/DaxStudio#943] [Q&A]
As the title says, I have SQL Server Profiler v19 installed and DAX Studio 3.0.0. When I go to Advanced -> SQL Profiler, the icon is greyed out and I can't click it, even when I'm connected a Power BI Model in Power BI Desktop. Any ideas or things I could try? I didn't see anything in the options to point DAX Studio to the SQL Server Profiler location or anything.

**Comment by dgosbell** [comment]
Which version of the SSMS v19 preview do you have installed? I have preview 3 installed on my machine and this feature is working OK.

The only reason this button would be greyed out is if DAX Studio cannot find a registered handler for profiler trace files. Can you maybe try re-installing v19?

DAX Studio checks the following registry key to find where profiler is installed
`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Classes\SQLServerProfilerTraceData\shell\open\command`

Can you check that on your machine and see if it shows something like the following?
![image](https://user-images.githubusercontent.com/8241830/190536842-3bbdc5d3-68d1-4eef-9d26-f135de781a0c.png)

**Reply by briannewtonpsyd** [reply]
That was the issue, thanks! I was missing that registry entry and reinstalling SSMS v19 seemed to create it.
